epub33/core/index.html

```diff
@@ -196,6 +196,18 @@ <h3>Organization</h3>
 			<section id="sec-intro-relations" class="informative">
 				<h3>Relationship to Other Specifications</h3>
 
+				<div class="caution">
+					<p>The technologies EPUB 3 builds on are constantly evolving. Some, typically referred to as
+						"living" or "evergreen" standards, are subject to change daily, while others may only be updated
+						to their latest versions when EPUB undergoes a new revision. In all cases, it is possible that
```

Comment:
The sentence sounds strange. It could be read as if those other standards are changed by EPUB...

Maybe saying "references to others"? Or say that those standards are subject to change rarely by a Working Group, typically only once in a few years.

Reply:
Ya, I've tried to clear this up in the latest commit and also pare down the caution a bit to make it more succinct.

See if it works better now.